compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/viewinterop/InteropView.uikit.kt

```diff
@@ -23,14 +23,11 @@ import androidx.compose.ui.semantics.semantics
 import androidx.compose.ui.uikit.utils.CMPInteropWrappingView
 import kotlinx.cinterop.readValue
 import platform.CoreGraphics.CGRectZero
+import platform.UIKit.UIResponder
 import platform.UIKit.UIView
 import platform.UIKit.UIViewController
 
-/**
- * On iOS, [InteropView] is a typealias for [UIView]. Interop entity can in fact be
- * a [UIViewController], in this case it will be wrapped in a [UIView] anyway.
- */
```

Comment:
The doc from public API was removed

Reply:
Because the doc is stale, let me update it.

Reply:
Done